contracts/core/SedaCoreV1.sol

```diff
@@ -141,62 +163,13 @@ contract SedaCoreV1 is
         _removePendingRequest(result.drId);
         delete _storageV1().requestDetails[result.drId];
 
-        // Fee distribution: handles three types of fees (request, result, batch)
-        // and manages refunds back to the requestor when applicable
-
-        // Amount to refund to requestor
-        uint256 refundAmount;
-
-        // Request fee distribution:
-        // - if invalid payback address, send all request fee to requestor
-        // - if valid payback address, split request fee proportionally based on gas used vs gas limit
-        if (requestDetails.requestFee > 0) {
-            address payableAddress = result.paybackAddress.length == 20
-                ? address(bytes20(result.paybackAddress))
-                : address(0);
-
-            if (payableAddress == address(0)) {
-                refundAmount += requestDetails.requestFee;
-            } else {
-                // Split request fee proportionally based on gas used vs gas limit
-                uint256 submitterFee = (result.gasUsed * requestDetails.requestFee) / requestDetails.gasLimit;
-                if (submitterFee > 0) {
-                    _transferFee(payableAddress, submitterFee);
-                    emit FeeDistributed(result.drId, payableAddress, submitterFee, ISedaCore.FeeType.REQUEST);
-                }
-                refundAmount += requestDetails.requestFee - submitterFee;
-            }
+        // If fee manager is not set, skip fee distribution
+        if (_storageV1().feeManager == IFeeManager(address(0))) {
+            return resultId;
         }
```

Comment:
Is there a reason we skip them now, instead of reverting with it no being set? With the old impl we didn't skip? Just me trying to understand :).

Reply:
Yea looks like old logic moved to the `_handleFeeDistribution` but we now don't execute it at all if the FeeManager contract isn't set.

Reply:
I just thought that there might be some weird case where we don't want to set a fee distribution and have a more altruistic setup.